deb-package/build

```diff
@@ -10,7 +10,7 @@ set -xeu
 
 # Set up package
 pkgname=libmxnet
-pkgversion_number=$(git describe --dirty |
+pkgversion_number=$(git describe --tags --dirty |
```

Comment:
I added this because not all upstream tags are annotated.  Obviously we can control the tags we apply, but I was getting frustrated at the builds of untagged commits coming out with versions like `0.9.3a+666` instead of `0.10.0+3`.

But I'll delete if you strongly prefer.